code/datums/spells/area_teleport.dm

```diff
@@ -78,15 +80,14 @@
 
 /obj/effect/proc_holder/spell/area_teleport/invocation(mob/user)
 	if(!invocation_area || !selected_area)
-		..()
-	else
-		switch(invocation_type)
-			if("shout")
-				user.say("[invocation] [uppertext(selected_area.name)]")
-				if(user.gender==MALE)
-					playsound(user.loc, pick('sound/misc/null.ogg','sound/misc/null.ogg'), 100, 1)
-				else
-					playsound(user.loc, pick('sound/misc/null.ogg','sound/misc/null.ogg'), 100, 1)
-			if("whisper")
-				user.whisper("[invocation] [uppertext(selected_area.name)]")
+		return
+	switch(invocation_type)
+		if("shout")
+			user.say("[invocation] [uppertext(selected_area.name)]")
+			if(user.gender==MALE)
+				playsound(user.loc, pick('sound/misc/null.ogg','sound/misc/null.ogg'), 100, 1)
+			else
+				playsound(user.loc, pick('sound/misc/null.ogg','sound/misc/null.ogg'), 100, 1)
```

Comment:
I might be blind but isnt this doing the same thing twice? what does the `null` sound even sound like?

Reply:
I just listened to that sound, its silence '-'

Reply:
Tested and no changes noted, wanna remove it while we are at it?

Reply:
please do